src/ui/activity-iframe-view.js

```diff
@@ -105,6 +110,21 @@ export class ActivityIframeView extends View {
     return this.shouldFadeBody_;
   }
 
+  /**
+   * Returns if the view shows loading indicator
+   * @return {boolean}
+   */
+  hasLoadingIndicator() {
+    return this.hasLoadingIndicator_;
+  }
+   /**
+   * Sets the loading indicator for this view
+   * @param {boolean} display
+   */
+  setLoadingIndicator(display) {
```

Comment:
So, not at all clear why we need a setter for this. Also, this var is declared above as a `@const`. Could you please explain?

Reply:
That was a copy paste error. @const is not necessary.

How else can we cache whether or not a view showed the loading indicator? The previousProgressView_ gets nulled out at the completion of view init.

Reply:
Let's discuss.